docker-compose.yml

```diff
@@ -3,7 +3,8 @@ services:
   cms:
     image: ${CMS_IMAGE}:${CMS_TAG}
     volumes:
-      - ./conf/cms/secrets.py:/code/taccsite_cms/secrets.py
+      - ./conf/cms/settings_secret.py:/code/taccsite_cms/settings_secret.py
+      - ./conf/cms/settings_custom.py:/code/taccsite_cms/settings_custom.py
```

Comment:
Hey, since this change isn't out for all portals yet, I'm adding it to the custom docker-compose file for cep core in dev for now. We'll probably merge this once all portals have the new cms setting

[docker-compose-overrides.dev.yml](https://github.com/TACC/Core-Portal-Deployments/commit/93f0a1d94ebd66194f7ea2ffc5f66fcb411b6102#diff-b2e433eaac8e1156615864e67e8ffa726e9cbfb406a79e62c9cf7cff2f25ca00)

Reply:
I think you could even reject it for now if you want.

The settings_secrets.py is just a file rename so that the cms container "secrets" are matched to the portal for naming consistency.

We are handling the settings_custom.py during the Jenkins CMS image build instead through the Core-CMS-Resources repo. That said, for the settings_custom piece to even work correctly here, we would need to add it to the Deployments repo and the filename would need to change to avoid conflict with the settings_custom.py file for the portal that already exists there.

This would be better to revisit after we have a chance to review the ways to integrate Deployments & Resources, things will change during that analysis and we really already solved this problem for the time being.